### ISSUE TITLE
How to handle phetioFeatured LinkedElements

### ISSUE THREAD

**Issue by zepumph** [phetsims/tandem#103]
From https://github.com/phetsims/gravity-force-lab/issues/174#issuecomment-524459433 two thoughts came up with @kathy-phet and I:

* It is annoying that you have regenerate the baseline after pasting in overrides to avoid an api mismatch. The is because if a model Property is marked as featured, then only after the baseline regeneration does the linkedElement also get marked as such
* We don't want the linked elements to be counted in the count in studio for the number of featured elements.

Here is one possible way forward:

Don't make linkedElements phetioFeatured in the baseline, and instead just add logic to studio that detects if something is a linked element, and if the thing its linking is featured, then it is displayed in the featured view. Here is a possible patch to make this happen. @samreid @chrisklus what do you think?

<details>

```diff
Index: studio/Studio.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- studio/Studio.js	(revision 6d48cabead79a1e232f757c0203cc31ae5a229b3)
+++ studio/Studio.js	(date 1566599036324)
@@ -212,11 +212,17 @@
      * @returns {boolean} - true if it passes the filters, and should be displayed
      */
     static shouldDisplayInTree( treeViewItem ) {
-      let ok = true;
       if ( phetioElementsDisplayType === featuredPhetioElementsRadioButton.value && !treeViewItem.metadata.phetioFeatured ) {
-        ok = false;
+        if ( treeViewItem.metadata.phetioTypeName === 'LinkedElementIO' ) {
+          assert && assert( typeof treeViewItem.stateObject.elementID === 'string',
+            'LinkedElementIO should have an elementID in state' );
+          if ( window.phetio.studio.instances[ treeViewItem.stateObject.elementID ].metadata.phetioFeatured ) {
+            return true;
+          }
+        }
+        return false;
       }
-      return ok;
+      return true;
     }
 
     /**

```
</details>

**Comment by samreid** [comment]
The idea seems reasonable, but I haven't had time to check the details of the proposed patch.   But if it works and addresses the API mismatch problem, it seems good to me to proceed.

**Comment by zepumph** [comment]
This is a duplicate of https://github.com/phetsims/phet-io-wrappers/issues/240. Closing
